lwm2m-client-ipso-presence.c

```diff
@@ -182,6 +182,18 @@ AwaError PresenceObject_DefineObjectsAndResources(AwaStaticClient *awaClient, Aw
     }
 
     AwaStaticClient_CreateObjectInstance(awaClient, IPSO_PRESENCE_OBJECT, 0);
+    error = AwaStaticClient_CreateResource(awaClient, IPSO_PRESENCE_OBJECT, 0, IPSO_PRESENCE_DIGITAL_INPUT_COUNTER_RESET);
+    if (error != AwaError_Success)
```

Comment:
It was not there earlier. i am getting 
 [ERROR] [lwm2m_client_core.c:1218] Read on 3302/0/5505 failed, handler returned 500 (expected 205)
at the startup of motion-sensor app.

however if i remove creation of IPSO_PRESENCE_DIGITAL_INPUT_COUNTER_RESET then do not get this warning.